src/main/java/com/mycompany/exercisethree/BasicCalculator.java

```diff
@@ -0,0 +1,57 @@
+package com.mycompany.exercisethree;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ *
+ * @author usuario
+ */
+public class BasicCalculator extends Calculator {
+
+    @Override
+    public void operateCalculator() {
+
+        Scanner getInt = new Scanner(System.in);
+        System.out.println("WELCOME TO THE BASIC CALCULATOR, PLEASE CHOOSE WHAT YOU WANT TO DO ");
+        System.out.println("1. ADD");
+        System.out.println("2. SUBSTRACT");
+        System.out.println("3. MULTIPLY");
+        System.out.println("4. DIVIDE");
+        System.out.println("5. RETURN");
+
+        try {
+            option = getInt.nextInt();
+            listNumbers = (option >= 1 && option <= 4) ? getInputString() : null;
```

Comment:
Please avoid using nullable this will cause to crash the app if no value pass, use and empty list instead

Reply:
Done.